spec/spec.md

```diff
@@ -595,7 +598,7 @@ Embedded or referenced test vectors.
 [[def: Sidetree]]
 ~ [Sidetree v1.0.0](https://identity.foundation/sidetree/spec/). Daniel Buchner, Orie Steele, Troy Ronda. 2021.03. Status: DIF Ratified Specification.
 
-[[def: Well Known DID]]
+[[def: Well Known DID,Well Known DIDs]]
```

Comment:
why?

Reply:
This will appear the same way in the html, as **just** `Well Known DID` within the references but then you can refer to it when you want as [[ref:WellKnownDIDs] and it will link correctly. Just a feature of spec-up